### PR TITLE
Adds an accent and several loadout items.

### PR DESCRIPTION
**Pull request by TheStryker**
Adds jumper dresses for more aesthetic choices with the underwear system (see picture), special thanks to NiennaB for the flavor texts, as well as a bucket hat.
Adds a colorable armband done by Mwahahahaha as well as an Idris armband.
Adds a Fisanduh accent written up by NewOriginalSchwann.
Adds a grey variant for the Himeo coat.

sprites loadout 3
showcase

### CODE REVIEW

**Comment by TheStryker** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/background/accent/human.dm [hunk 1]
```diff
@@ -143,4 +143,11 @@
 	description = "The so-called \"Deep Frontier\" accent is not truly a unified accent at all, and instead defines a broad range of accents from the more \"wild\" frontier outside of human control. \
 	Those with these accents are almost universally looked down upon as wild savages by their counterparts in more civilized space at best. Speakers of this accent are almost always found in lower-end \
 	positions in civilized space, due to the typically impoverished and anarchic nature of the Deep Frontier."
-	tag_icon = "deepfrontier"
\ No newline at end of file
+	tag_icon = "deepfrontier"
+
+/datum/accent/fisanduh
+	name = ACCENT_FISANDUH
+	description = "While it is similar to the typical Morozian accent to those not familiar with, or from, the Empire of Dominia the standard Fisanduhian accent has some variations that mark it as distinctive. \
```
Comment (alsoandanswer) [inline code/modules/background/accent/human.dm:150]:
```suggestion
	description = "While it is similar to the typical Morozian accent, to those not familiar with it; or from the Empire of Dominia, the standard Fisanduhian accent has some variations that mark it as distinctive. \
```

**Comment by alsoandanswer** [comment]
do you also mind fixing the shading on the skirts to be in line with the rest

also add inhands

**Comment by TheStryker** [comment]
> do you also mind fixing the shading on the skirts to be in line with the rest
> 
> also add inhands

image


Updated. Inhands have also been added, derived from the tube inhands.

**Review by Geevies — APPROVED** [review]

**Pull request merged**